src/routes/index.ts

```diff
@@ -1,9 +1,11 @@
 import express from "express";
 
 import router from "./serverRoute";
+import authRouter from "./apiAuth.route";
 
 const routes = express();
 
-routes.use("/api", router);
+routes.use("/api/v1", router);
+routes.use("/api/v1", authRouter);
```

Comment:
Why two pointing to the send endpoint?

Reply:
The first endpoint is: /api/v1, while the second endpoint is: /api/v1/auth